### ISSUE TITLE
Wallet crash importing ZCL private keys

### ISSUE THREAD

**Issue by anibol** [BTCPrivate/electrum-btcp#20]
Hi, i'm on Windows with BTCP-Electrum Wallet. I have exported my private keys from ZCL-Electrum Wallet (is multisig) and when trying to import to a new wallet on BTCP Wallet the app crash. Any help? Thanks

**Comment by ch4ot1c** [comment]
Yes - the addresses don't match because they're changed between the two changes. We are actively working on this.
